app/views/proposal_forms/_proposal_fields.html.erb

```diff
@@ -7,6 +7,8 @@
         <%=  render partial: 'proposal_fields/radio', locals: { field: field } %>
       <% elsif field.fieldable_type.eql?('ProposalFields::SingleChoice') %>
         <%= render partial: 'proposal_fields/single_choice', locals: { field: field } %>
+      <% elsif field.fieldable_type.eql?('ProposalFields::Date') %>
+        <%= render partial: 'proposal_fields/date', locals: { field: field } %>
       <% else %>
         <%= render partial: 'proposal_fields/multi_choice', locals: { field: field } %>
       <% end %>
```

Comment:
Can we move this block of conditionals to a helper, please?

In the helper, instead of a bunch of if/else statements, you could do:
```
<% proposal_fields&.each do |field| %>
  <%= render partial: field.fieldable_type.split('::').last.underscore
<% end %>
```